akka-http-backend/src/main/scala/sttp/client3/akkahttp/AkkaHttpServerSentEvents.scala

```diff
@@ -0,0 +1,17 @@
+package sttp.client3.akkahttp
+
+import akka.NotUsed
+import akka.stream.scaladsl.{Flow, Framing}
+import akka.util.ByteString
+import sttp.client3.sse.ServerSentEvent
+
+object AkkaHttpServerSentEvents {
+
+  def decodeSSE: Flow[ByteString, ServerSentEvent, NotUsed] =
+    //todo decide on max frame length
+    Framing
+      .delimiter(ByteString("\n\n"), maximumFrameLength = Int.MaxValue, allowTruncation = true)
```

Comment:
`maximumFrameLength` can be parameter accepted by `decodeSSE` method

Reply:
Is there anything specified by the SSE standard? Maximum message size? If not, we should make this configurable, with some reasonable default - maybe taken from other impls?